compiler/src/dmd/backend/cdef.d

```diff
@@ -596,7 +596,7 @@ struct Configv
 }
 
 alias reg_t = ubyte;            // register number
-alias regm_t = uint;            // Register mask type
+alias regm_t = ulong;           // Register mask type
```

Comment:
```
src/dmd/backend/x86/cod4.d(729): Error: function `dmd.backend.code.findreglsw(uint regm)` is not callable using argument types `(ulong)`
src/dmd/backend/x86/cod4.d(729):        cannot pass argument `retregs` of type `ulong` to parameter `uint regm`
src/dmd/backend/x86/cod4.d(759): Error: function `dmd.backend.code.findregmsw(uint regm)` is not callable using argument types `(ulong)`
src/dmd/backend/x86/cod4.d(759):        cannot pass argument `retregs` of type `ulong` to parameter `uint regm`
src/dmd/backend/x86/cod4.d(787): Error: function `dmd.backend.code.findreglsw(uint regm)` is not callable using argument types `(ulong)`
src/dmd/backend/x86/cod4.d(787):        cannot pass argument `retregs` of type `ulong` to parameter `uint regm`
src/dmd/backend/x86/cod4.d(1235): Error: function `dmd.backend.code.findregmsw(uint regm)` is not callable using argument types `(ulong)`
src/dmd/backend/x86/cod4.d(1235):        cannot pass argument `retregs` of type `ulong` to parameter `uint regm`
src/dmd/backend/x86/cod4.d(1236): Error: function `dmd.backend.code.findreglsw(uint regm)` is not callable using argument types `(ulong)`
src/dmd/backend/x86/cod4.d(1236):        cannot pass argument `retregs` of type `ulong` to parameter `uint regm`
src/dmd/backend/x86/cod4.d(1259): Error: function `dmd.backend.code.findreglsw(uint regm)` is not callable using argument types `(ulong)`
src/dmd/backend/x86/cod4.d(1259):        cannot pass argument `retregs` of type `ulong` to parameter `uint regm`
src/dmd/backend/x86/cod4.d(1263): Error: function `dmd.backend.code.findregmsw(uint regm)` is not callable using argument types `(ulong)`
src/dmd/backend/x86/cod4.d(1263):        cannot pass argument `retregs` of type `ulong` to parameter `uint regm`
src/dmd/backend/x86/cod4.d(1329): Error: function `dmd.backend.code.findreglsw(uint regm)` is not callable using argument types `(ulong)`
src/dmd/backend/x86/cod4.d(1329):        cannot pass argument `retregs` of type `ulong` to parameter `uint regm`
src/dmd/backend/x86/cod4.d(1357): Error: function `dmd.backend.code.findreglsw(uint regm)` is not callable using argument types `(ulong)`
src/dmd/backend/x86/cod4.d(1357):        cannot pass argument `retregs` of type `ulong` to parameter `uint regm`
src/dmd/backend/x86/cod4.d(1642): Error: function `dmd.backend.code.findreglsw(uint regm)` is not callable using argument types `(ulong)`
src/dmd/backend/x86/cod4.d(1642):        cannot pass argument `rretregs` of type `ulong` to parameter `uint regm`
src/dmd/backend/x86/cod4.d(1643): Error: function `dmd.backend.code.findregmsw(uint regm)` is not callable using argument types `(ulong)`
src/dmd/backend/x86/cod4.d(1643):        cannot pass argument `rretregs` of type `ulong` to parameter `uint regm`
src/dmd/backend/x86/cod4.d(1650): Error: function `dmd.backend.global.mask(uint m)` is not callable using argument types `(ulong)`
src/dmd/backend/x86/cod4.d(1650):        cannot pass argument `rhi` of type `ulong` to parameter `uint m`
src/dmd/backend/x86/cod4.d(1651): Error: function `dmd.backend.x86.code_x86.modregrm(uint m, uint r, uint rm)` is not callable using argument types `(int, ulong, int)`
src/dmd/backend/x86/cod4.d(1651):        cannot pass argument `rhi` of type `ulong` to parameter `uint r`
src/dmd/backend/x86/cod4.d(1652): Error: function `dmd.backend.x86.code_x86.modregrm(uint m, uint r, uint rm)` is not callable using argument types `(int, int, ulong)`
src/dmd/backend/x86/cod4.d(1652):        cannot pass argument `rlo` of type `ulong` to parameter `uint rm`
src/dmd/backend/x86/cod4.d(1653): Error: function `dmd.backend.x86.code_x86.modregrm(uint m, uint r, uint rm)` is not callable using argument types `(int, ulong, int)`
src/dmd/backend/x86/cod4.d(1653):        cannot pass argument `rhi` of type `ulong` to parameter `uint r`
src/dmd/backend/x86/cod4.d(1654): Error: function `dmd.backend.x86.code_x86.modregrm(uint m, uint r, uint rm)` is not callable using argument types `(int, int, ulong)`
src/dmd/backend/x86/cod4.d(1654):        cannot pass argument `rlo` of type `ulong` to parameter `uint rm`
src/dmd/backend/x86/cod4.d(1655): Error: function `dmd.backend.x86.code_x86.modregrm(uint m, uint r, uint rm)` is not callable using argument types `(int, int, ulong)`
src/dmd/backend/x86/cod4.d(1655):        cannot pass argument `rhi` of type `ulong` to parameter `uint rm`
src/dmd/backend/x86/cod4.d(1663): Error: function `dmd.backend.code.findregmsw(uint regm)` is not callable using argument types `(ulong)`
src/dmd/backend/x86/cod4.d(1663):        cannot pass argument `retregs` of type `ulong` to parameter `uint regm`
src/dmd/backend/x86/cod4.d(1663): Error: function `dmd.backend.code.findreglsw(uint regm)` is not callable using argument types `(ulong)`
src/dmd/backend/x86/cod4.d(1663):        cannot pass argument `retregs` of type `ulong` to parameter `uint regm`
src/dmd/backend/x86/cod4.d(2246): Error: function `dmd.backend.code.findregmsw(uint regm)` is not callable using argument types `(ulong)`
```

Reply:
I know, this change is exposing several typing errors in the backend!